anesthetic/plot.py

```diff
@@ -39,6 +40,16 @@
 class AxesSeries(pandas.Series):
     """Anesthetic's axes version of `pandas.Series`."""
 
+    def __init__(self, data=None, index=None, fig=None, ncol=None, labels=None,
+                 gridspec_kw=None, subplot_spec=None, *args, **kwargs):
+        if data is None and index is not None:
+            data = pandas.Series(np.full(np.shape(index), None), index=index)
```

Comment:
This could be replaced with `super().__init__(np.full(np.shape(index), None), index)`